lib/msal-browser/docs/testing.md

```diff
@@ -0,0 +1,143 @@
+# Testing
+
+MSAL Browser starting version 2.17.0 has added the `loadTokens()` API, which allows the loading of id tokens and access tokens to the MSAL cache, which can then be read out using `acquireTokenSilent()`. 
+
+**Note: This is an advanced feature that is intended for testing purposes only. Loading tokens to your application's cache may cause your app to break.**
+
+## The loadTokens() API
+
+The `loadToken()` API can be accessed by calling `getTokenCache()` on MSAL Browser's `PublicClientApplication` instance. 
+
+```js
+myMSALObj.getTokenCache().loadTokens(silentRequest, serverResponse, loadTokenOptions);
+```
+
+`loadTokens()` takes in a request of type `SilentRequest`, a response of type `ServerAuthenticationROPCRespones`, and options of type `LoadTokenOptions`.
+
+See the type definitions for each below:
+
+- [`SilentRequest`](https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_browser.html#silentrequest)
+- [`ServerAuthenticationROPCResponse`](https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_common.html#serverauthorizationropcresponse)
```

Comment:
Do we need to add how to fetch this or is it already covered?

Reply:
Thanks for pointing that out. I added instructions for importing.